lua-5.1/src/lua.h

```diff
@@ -9,6 +9,16 @@
 #ifndef lua_h
 #define lua_h
 
+#ifdef WIN32
+# ifndef LUA_WIN
+#  define LUA_WIN
+# endif
+# ifndef _WIN32
```

Comment:
would be cleaner to not mess wit hthe lua sourcecode I reckon?  I think I'd prefer these in our CMakeLists perhaps?